src/main/java/com/prgrms/needit/domain/user/openAPI/service/OpenApiService.java

```diff
@@ -0,0 +1,129 @@
+package com.prgrms.needit.domain.user.openAPI.service;
+
+import com.fasterxml.jackson.core.JacksonException;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.prgrms.needit.common.error.ErrorCode;
+import com.prgrms.needit.common.error.exception.InvalidArgumentException;
+import com.prgrms.needit.common.error.exception.OpenApiException;
+import com.prgrms.needit.domain.user.openAPI.dto.BusinessRawResponse;
+import com.prgrms.needit.domain.user.openAPI.dto.BusinessRefinedResponse;
+import com.prgrms.needit.domain.user.openAPI.dto.BusinessRequest;
+import com.prgrms.needit.domain.user.openAPI.dto.BusinessesRequest;
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.io.OutputStreamWriter;
+import java.io.UnsupportedEncodingException;
+import java.net.HttpURLConnection;
+import java.net.URL;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.PropertySource;
+import org.springframework.stereotype.Service;
+
+@Slf4j
+@Service
+@PropertySource("classpath:application-dev.yml")
```

Comment:
`release/NEED-250` 으로 머지할때는 
```suggestion
@PropertySource("classpath:application.yml")
```
으로 수정해서 머지해주세요!

Reply:
네엡!!👩‍🔧